src/components/UI/VersionPicker/VersionPicker.js

```diff
@@ -0,0 +1,220 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import DropdownMenu from 'UI/DropdownMenu';
+
+// AppVersionPicker takes a list of app versions and a currently selected version
```

Comment:
I believe it would make more sense to have this comment right on top of the `AppVersionPicker` component.

Also, a topic for discussion would be switching to only using JSDoc style comments, instead of using a mix between JSDoc and Golang comments. But that's not in the scope of this PR

Reply:
I'll bring this up in the next chapter sync. I'd think JSDoc style is the way to go. And i'll make a change here in this PR in the meanwhile.